BUILD.ewdk.toolchains.tpl

```diff
@@ -195,7 +195,7 @@ ewdk_cc_toolchain_config(
 )
 
 toolchain(
-    name = "ewdk-cc-toolchain-x64_windows",
+    name = "ewdk-cc-toolchain-x64_x64_windows",
```

Comment:
Wasn't quite sure of the naming convention here. I went with being explicit always instead of compressing the cases of `x64_x64` -> `x64` and `arm64_arm64` -> `arm64`. Could go either way.